components/Modal.js

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
```

Comment:
Importing such a library is an heavy job 😅  
Import just what is necessary - in this case, `useState`.